extensions/blocks/contact-form/components/jetpack-crm-connection-settings.js

```diff
@@ -92,11 +96,15 @@ function CRMConnectionSettings( props ) {
 		// Either no valid version or Jetpack CRM is not installed.
 		return (
 			<p>
```

Comment:
It looks like I forgot to add the `jetpack-contact-form__crm_text` class to this element in my original commit. I'll fix that.

Reply:
See commit 5e433a0.